src/mpyl/cli/build.py

```diff
@@ -135,51 +138,88 @@ def run(obj: CliContext, ci, all_, tag):  # pylint: disable=invalid-name
 
 
 @build.command(help="The status of the current local branch from MPyL's perspective")
+@click.option(
+    "-f",
+    "--fix-pr",
```

Comment:
Can you explain your chain of thought for implementing a toggle instead of just always making sure everything is in the state needed for the logic that follows or always fetching the branch/checking out main (if that hasn't been done yet) when starting mpyl?